### PR TITLE
Address user/role dup.

### PR DESCRIPTION
**Pull request by stbof**
DOC-728

This issue was discussed a year ago, but not resolved. Requesting reviews from discussion participants.

I tried removing the `*USER` pages, but it turned out to be complicated because so many referring pages were assuming a link to a `*USER` page. This is a different approach to resolving the issue with SQL diagrams and duplication of role option lists.

### CODE REVIEW

**Review by ianjevans — APPROVED** [review]

**Comment by knz** [comment]
I'm too far removed from these aspects nowadays and will trust @RichardJCai for the technical review.

File: v22.2/alter-user.md [hunk 1]
```diff
@@ -2,26 +2,24 @@
...
 
 ## Required privileges
 
- To alter other users, the user must be a member of the `admin` role or have the [`CREATEROLE`](create-user.html#create-a-user-that-can-create-other-users-and-manage-authentication-methods-for-the-new-users) parameter set.
+To alter other users, the user must be a member of the `admin` role or have the [`CREATEROLE`](create-role.html#create-a-role-that-can-create-other-roles-and-manage-authentication-methods-for-the-new-roles) [role option](#role-options).
 
 ## Synopsis
 
-<div>{% include {{ page.version.version }}/sql/generated/diagrams/alter_user_password.html %}</div>
+See [Synopsis](alter-role.html#synopsis).
```
Comment (nickvigilante) [inline v22.2/alter-user.md:22]: Nit: Should this be See Alter Role instead of See Synopsis?

**Review by nickvigilante — APPROVED** [review]
One nit that you can take or leave. LGTM!

File: v22.2/alter-user.md (hunk 1 above)
Comment (stbof) [reply v22.2/alter-user.md:22]: I guess each one could say See *Role: Synopis, where * is Alter, Create, and Drop.

**Review by RichardJCai — APPROVED** [review]
:LGTM:

**Pull request merged**